tests/cases/struct/LV_NeoHookean_passive_genBC/svFSI.inp

```diff
@@ -0,0 +1,94 @@
+# run with https://github.com/vvedula22/svFSI
+#----------------------------------------------------------------
+# General simulation parameters
+
+Continue previous simulation: f
+Number of spatial dimensions: 3
+Number of time steps: 100
```

Comment:
Same here (and for all the other test cases)

Reply:
Confirming that all `.inp` and `.xml` have exactly the same parameters.

Reply:
Following in person discussion with Matteo and Martin, removing all svFSI.inp files from test cases